vite.base.config.ts

```diff
@@ -55,13 +56,14 @@ export function getBuildDefine(env: ConfigEnv<'build'>) {
   const defineKeys = getDefineKeys(names)
   const define = Object.entries(defineKeys).reduce(
     (acc, [name, keys]) => {
-      const { VITE_DEV_SERVER_URL, VITE_NAME } = keys
+      const { VITE_DEV_SERVER_URL, VITE_NAME, SENTRY_DSN } = keys
       const def = {
         [VITE_DEV_SERVER_URL]:
           command === 'serve'
             ? JSON.stringify(process.env[VITE_DEV_SERVER_URL])
             : undefined,
         [VITE_NAME]: JSON.stringify(name),
+        [SENTRY_DSN]: JSON.stringify(process.env.SENTRY_DSN),
```

Comment:
We need the Sentry DSN variable to be statically replaced during build time since env variables are not available during runtime in Prod.